kaniko-executer/Dockerfile

```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Builds the static Go image to execute in a Kubernetes job
+# Builds the static Go image (1.18.1-alpine3.15) to execute in a Kubernetes job
 
-FROM eu.gcr.io/kyma-project/external/golang:1.18.1-alpine3.15 as builder
+# image builder base on golang:1.19.0-alpine3.16
+FROM golang@sha256:78767a599f5758e3fd60fd065de9dc40aec1ad3b86ded2bddc92640a465cb515 as builder
```

Comment:
Why do we switch from `eu.gcr.io` to docker hub?
I can't find this sha, this version was release 12 days ago:
https://hub.docker.com/layers/golang/library/golang/1.19.0-alpine3.16/images/sha256-70df3b8f9f099da7f60f0b32480015165e3d0b51bfacf9e255b59f3dd6bd2828?context=explore

Reply:
the lack of `eu.gcr.io/kyma-project/external/golang` registry it a copy/paste error; will fix that

Reply:
thx